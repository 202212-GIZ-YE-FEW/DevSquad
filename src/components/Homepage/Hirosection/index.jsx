import Image from "next/image";
import { useTranslation } from "next-i18next";
import { useEffect } from "react";
import AOS from "aos";
import "aos/dist/aos.css";
import Getstartedbtn from "@/components/Getstartedbtn";

const Hirosection = () => {
    const { t } = useTranslation("common");
    useEffect(() => {
        AOS.init();
    }, []);
    return (
        <div className='container mx-auto px-6 pt-12	pb-4 grid lg:grid-cols-2 gap-10  grid-cols-1 font-Rubik'>
            <div
                className='relative flex flex-col align-content-center justify-center align-items-start text-center lg:text-start'
                data-aos='fade-right'
            >
                <div className='md:absolute top-14 left-0 right-0 bottom-0 static'>
                    <div className='flex flex-col md:pt-20 pt-5'>
                        <p className='text-4xl font-medium p-2'>
                            {t("home.hiro.title")}
                        </p>
                        <p className='text-2xl text-primary-gray px-2 font-normal'>
                            {t("home.hiro.describtion")}
                        </p>
                    </div>
                    <div className='pt-10	'>
                        <Getstartedbtn />
                    </div>
                </div>
            </div>
            <div className='overflow-hidden' data-aos='fade-left'>
                <Image
                    src='/images/Image.png'
                    alt='hiro'
                    width={638}
                    height={638}
                    className='object-cover rounded-xl'
                />
            </div>
        </div>
    );
};

export default Hirosection;
