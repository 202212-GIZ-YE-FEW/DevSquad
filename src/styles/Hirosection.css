/* .imagewidth{
  width: 638px;
  height: 638px;
} */
