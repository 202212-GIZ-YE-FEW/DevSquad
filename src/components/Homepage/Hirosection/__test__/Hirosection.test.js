import renderer from "react-test-renderer";
import Hirosection from "../Index";

it("renders correctly", () => {
    const tree = renderer.create(<Hirosection />).toJSON();
    expect(tree).toMatchSnapshot();
});
