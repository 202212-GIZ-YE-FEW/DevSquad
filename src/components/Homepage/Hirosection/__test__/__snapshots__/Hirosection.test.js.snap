// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container mx-auto md:px-32 px-10 pt-12 pb-4 grid md:grid-cols-2 gap-10  sm:grid-cols-1 font-Rubik"
>
  <div
    className="flex flex-col align-content-center justify-center align-items-start text-center md:text-left"
  >
    <div
      className="flex flex-col "
    >
      <p
        className="text-5xl font-medium"
      >
        title
      </p>
      <p
        className="text-2xl text-primary-gray px-2 font-normal"
      >
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Consectetur at, illum itaque facilis minima iusto reiciendis ipsum veniam optio perspiciatis.
      </p>
    </div>
    <div
      className="pt-10 "
    >
      <a
        className="w-20 h-7 py-2.5 px-4 rounded-lg bg-primary-blue text-white text-2xl"
        href="/blogs"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        Get Started
      </a>
    </div>
  </div>
  <div
    className="relative h-96 overflow-hidden"
  >
    <span
      style={
        Object {
          "background": "none",
          "border": 0,
          "bottom": 0,
          "boxSizing": "border-box",
          "display": "block",
          "height": "initial",
          "left": 0,
          "margin": 0,
          "opacity": 1,
          "overflow": "hidden",
          "padding": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
          "width": "initial",
        }
      }
    >
      <img
        alt="profile"
        className="object-cover rounded-xl"
        data-nimg="fill"
        decoding="async"
        onError={[Function]}
        onLoad={[Function]}
        src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
        style={
          Object {
            "border": "none",
            "bottom": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": 0,
            "left": 0,
            "margin": "auto",
            "maxHeight": "100%",
            "maxWidth": "100%",
            "minHeight": "100%",
            "minWidth": "100%",
            "objectFit": undefined,
            "objectPosition": undefined,
            "padding": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "width": 0,
          }
        }
      />
      <noscript>
        <img
          alt="profile"
          className="object-cover rounded-xl"
          data-nimg="fill"
          decoding="async"
          loading="lazy"
          sizes="100vw"
          src="/_next/image?url=%2Fimages%2FImage.png&w=3840&q=75"
          srcSet="/_next/image?url=%2Fimages%2FImage.png&w=640&q=75 640w, /_next/image?url=%2Fimages%2FImage.png&w=750&q=75 750w, /_next/image?url=%2Fimages%2FImage.png&w=828&q=75 828w, /_next/image?url=%2Fimages%2FImage.png&w=1080&q=75 1080w, /_next/image?url=%2Fimages%2FImage.png&w=1200&q=75 1200w, /_next/image?url=%2Fimages%2FImage.png&w=1920&q=75 1920w, /_next/image?url=%2Fimages%2FImage.png&w=2048&q=75 2048w, /_next/image?url=%2Fimages%2FImage.png&w=3840&q=75 3840w"
          style={
            Object {
              "border": "none",
              "bottom": 0,
              "boxSizing": "border-box",
              "display": "block",
              "height": 0,
              "left": 0,
              "margin": "auto",
              "maxHeight": "100%",
              "maxWidth": "100%",
              "minHeight": "100%",
              "minWidth": "100%",
              "objectFit": undefined,
              "objectPosition": undefined,
              "padding": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": 0,
            }
          }
        />
      </noscript>
    </span>
  </div>
</div>
`;
